README.md

```diff
@@ -116,7 +119,31 @@ endpoints:
 
 If the `POSTGRES_CONNECTION_STRING` environment variable is set, PADS will connect to the specified Postgres database.
 
-<!-- TODO(@commoddity) - add further details on Postgres implementation -->
+#### 3.2.1. Grove Portal DB Compatibility
+
+The [Postgres Driver schema file](postgres/sqlc/schema.sql) uses tables from the existing Grove Portal database schema, allowing PATH to source its authorization data from the existing Grove Portal DB.
```

Comment:
Make sure to rename these files as well after the changes elsewhere are tended to.